go.mod

```diff
@@ -3,11 +3,10 @@ module github.com/umputun/tg-spam
 go 1.23
 
 require (
-	github.com/OvyFlash/telegram-bot-api v0.0.0-20241107191146-851f2334eccf
```

Comment:
Apologies for introducing this problem, but this line is unchanged from the current master, so I presume you need to rebase this branch against master for it to go away from the diff.